golink.go

```diff
@@ -69,6 +70,8 @@ var localClient *tailscale.LocalClient
 func Run() error {
 	flag.Parse()
 
+	hostinfo.SetPackage("golink")
```

Comment:
Hmm, is this a package name? Maybe this should be a new "app" field or something, to tell it apart from deb/rpm/snap/etc. ?

Reply:
Done. PTAL